From d7b0aa0df41d24fc1875a8d87c9298eb9098cf09 Mon Sep 17 00:00:00 2001
From: Robert Ogden <robertogden@chromium.org>
Date: Mon, 13 Sep 2021 19:23:58 +0000
Subject: [PATCH] Missing header when TFLite isn't built in

Change-Id: I4e89f7b8f2d67343e219a1935dbdcb64682a5df9
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3158501
Auto-Submit: Robert Ogden <robertogden@chromium.org>
Commit-Queue: Michael Crouse <mcrouse@chromium.org>
Reviewed-by: Michael Crouse <mcrouse@chromium.org>
Cr-Commit-Position: refs/heads/main@{#920867}

--- a/components/optimization_guide/content/browser/page_content_annotations_service.cc
+++ b/components/optimization_guide/content/browser/page_content_annotations_service.cc
@@ -6,6 +6,7 @@
 
 #include "base/metrics/histogram_functions.h"
 #include "base/strings/string_util.h"
+#include "base/strings/utf_string_conversions.h"
 #include "components/history/core/browser/history_service.h"
 #include "components/optimization_guide/core/optimization_guide_enums.h"
 #include "components/optimization_guide/core/optimization_guide_features.h"
